README.md

```diff
@@ -1406,6 +1407,12 @@ The IOS operating system does not support:
 * enable
 * retransmit_count
 
+#### ios_additional_syslog_settings
+
+##### 4948
+
+The `logging origin-id` command is not avalible on th above machine.
```

Comment:
This should read: `origin_id` attribute is not available on the above device.

Reply:
fixed